README.md

```diff
@@ -1,16 +1,85 @@
 # Tech Tests
 
-This repository works a question bank that volunteers can use for conducting mock interviews.
+This repository works as a question bank that volunteers can use for conducting mock interviews.
 
-## How to use it
+## How do the Tests Comprise a Mock Interview?
+
+For mock interviews to be effective, you have to expose the candidates to as many potential assessment methods as possible, so that they are well prepared for the real thing.
+
+Part of this will involve a well-conducted technical challenge in order to assess the following characteristics:
+
+* A candidate's approach to finding a solution to the problem
+* A candidate's ability to design a solution to the problem and describe the design
+* A candidate's approach to test-driven design and development
+* The quality of the implemented solution itself
+
+A mistake often made is just focussing on the last point, but employers of good software engineers will look at all of the above.
+
+Whan assessing a candidate's attempt at one of these tests, be sure to assess and give feedback on all of the above.  There will be other questions and exercises beyond these tech tests too.  **The aim of this is to ensure all technical tests are conducted consistently during mock interviews.**
+
+
+## How to use these Tests
 
 1. Pick one of the top-level folders for the category that best describes your question type:
     - data-structures-and-algorithms - Usually language agnostic. Focus more on correct use of logic, algorithms and data structures
     - homework-assignment - An assignment to be given to the candidate to be completed offline
-    - live-assignment - A task for the candidate to complete in a live coding session
-2. Create a new folder for your question with a descriptive name inside the category folder
-3. Inside your question folder, create a folder named `exercise`. A zip version of this folder will be given to the interviewee
-4. Inside the `exercise` folder, add a `README.md` file. Ideally the file should contain:
+    - live-assignment - A task for the candidate to complete in a live coding session or mock interview
+1. Within the folder chosen in 1. there will be some question folders, each of which contains a different technical test.
```

Comment:
On these, I'm pretty sure with Markdown you can just have 1. all the way down and it renders a numbered list.  Just makes it easier than having to manually renumber as the list is modified.

Reply:
Is that part of the standard? I checked some markdown documentations and none of them mentions that you can use `1` and it renders correctly.

Reply:
To be fair, the manual says the numbers aren't important and you still get 1, 2, 3...n regardless of whether it's 1. 1. 1. or 3. 8. 2. or whatever, so I suppose it's down to personal preference.

Reply:
I have never seen that, but I wonder if it's standard and respected by all editors. Can you point me to the place where you found that information?

Reply:
There's [this answer on SO](https://stackoverflow.com/questions/50642050/markdown-how-to-create-ordered-lists) which points to some (supposedly) [official documentation](https://daringfireball.net/projects/markdown/syntax#list).  In any case, it definitely works on GitHub.

Reply:
Thanks for the link. It makes sense since the HTML doesn't include the numbers. I think it's still a good idea to use the right numbers since people might be reading the markdown file directly

Reply:
Alright, as you wish.  I've always favoured the low maintenance option, but pros and cons to both, I suppose.